src/PlexShareScreenshare/Client/ScreenProcessor.cs

```diff
@@ -0,0 +1,224 @@
+///<author>Satyam Mishra</author>
+///<summary>
+/// This file has ScreenProcessor class. It is responsible for 
+/// processing the image from ScreenCapturer class and calculating
+/// the image bits that are different from the previous image
+///</summary>
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Threading;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.Runtime.InteropServices;
+
+// This datatype is for storing a list of the coordinate
+// of pixel and its RGB values i.e. list of ((x, y), (R, G, B))
+using ImageDiffList = System.Collections.Generic.List<System.Tuple<System.Tuple<int, int>,
+                        System.Tuple<int, int, int>>>;
+
+// Each frame consists of the resolution of the image and the ImageDiffList
+using Frame = System.Tuple<System.Tuple<int, int>,
+                        System.Collections.Generic.List<System.Tuple<System.Tuple<int, int>,
+                        System.Tuple<int, int, int>>>>;
+
+namespace PlexShareScreenshare.Client
+{
+    /// <summary>
+    /// Class contains implementation of the screen processing using threads (tasks)
+    /// </summary>
+    internal class ScreenProcessor
+    {
+        // The queue in which the image will be enqueued after
+        // processing it
+        private Queue<Frame> ProcessedFrame;
```

Comment:
private members should start with underscore and small letter. This should be _processedFrame